### PR TITLE
Fix the typo in document configs.rst

### PR DESCRIPTION
**Pull request by jinchengchenghh**

### CODE REVIEW

**Comment by jinchengchenghh** [comment]
Could you take a look? @Yuhta

**Comment by jinchengchenghh** [comment]
Can you help review? Thanks! @xiaoxmeng

File: velox/docs/configs.rst [hunk 1]
```diff
@@ -280,7 +280,7 @@ Hive Connector
      - bool
      - false
      - True if reading the source file column names as lower case, and planner should guarantee
-     - the input column name and filter is also lower case to achive case-insensitive read..
```
Comment (laithsakka) [inline velox/docs/configs.rst:283]: do we need one . after read?

**Review by laithsakka — APPROVED** [review]

File: velox/docs/configs.rst (hunk 1 above)
Comment (jinchengchenghh) [reply velox/docs/configs.rst:283]: Removed, can we merge it? Thanks!

**Comment by jinchengchenghh** [comment]
Can we merge it? Thanks! @laithsakka

**Pull request closed without merging**